message-queue-lib/src/cmr/message_queue/services/queue.clj

```diff
@@ -37,6 +37,8 @@
     (Thread/sleep (config/messaging-retry-delay))
     (recur queue-broker exchange-name msg)))
 
+(declare publish-message)
+(declare queue-broker exchange-name msg)
```

Comment:
Not sure if you need to declare queue-broker, exchange-name and msg

Reply:
o I don't need to at all, but the linter can't find these values and complains. I think this is one of the reasons why people stopped using the linter. I don't know what to do here other than just include these. Any suggestions?

Reply:
I don't have any alternatives, so I am fine with this